README.md

```diff
@@ -94,26 +146,10 @@ Feel free to check the [issues page](../../issues/).
 
 > Write a message to encourage readers to support your project
 
-If you like this project please let me know
-
+If you like this project...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank my coding partners for their helps
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
-
```

Comment:
- [ ] Great job so far @IsmailChangezi  Sorry but I have to request changes to ensure you have met all the requirements of the previous reviewers. Here I think you misunderstood what the previous reviewer meant. He only meant you should remove the instructions in your README that starts with the greater than symbol `>` eg. 
![image](https://user-images.githubusercontent.com/102699958/227727097-bd083baa-d578-445f-9390-92643896da55.png)
- [ ] Currently, you've removed the `Acknowledgments section` as well but as the previous reviewer mentioned, all the headers in the README template are compulsory with the exception of the `FAQ` section so kindly Include the Acknowledgements section :100: